code/model/editableformfields/EditableFormField.php

```diff
@@ -604,7 +627,7 @@ public function doUpdateFormField($field) {
 	 */
 	protected function updateFormField($field) {
 		// set the error / formatting messages
-		$field->setCustomValidationMessage($this->getErrorMessage());
+		$field->setCustomValidationMessage($this->getErrorMessage()->RAW());
```

Comment:
Because core validation will internally XML encode this value, so don't return the encoded value here.